packages/nimble-tokens/source/styledictionary/build.js

```diff
@@ -3,8 +3,6 @@ require('./nimble-extensions');
 const StyleDictionary = require('style-dictionary');
 
 (async () => {
```

Comment:
ooo I noticed it but likes the linter found it too, no reason for this to be async anymore